code/datums/components/cult_ritual_item.dm

```diff
@@ -300,12 +300,14 @@
 			return
 		our_turf = get_turf(cultist) //we may have moved. adjust as needed...
 
+	var/cached_blood_volume = cultist.get_blood_volume()
+
 	cultist.visible_message(
-		span_warning("[cultist] [cultist.blood_volume ? "cuts open [cultist.p_their()] arm and begins writing in [cultist.p_their()] own blood":"begins sketching out a strange design"]!"),
-		span_cult("You [cultist.blood_volume ? "slice open your arm and ":""]begin drawing a sigil of the Geometer.")
+		span_warning("[cultist] [cached_blood_volume ? "cuts open [cultist.p_their()] arm and begins writing in [cultist.p_their()] own blood":"begins sketching out a strange design"]!"),
+		span_cult("You [cached_blood_volume ? "slice open your arm and ":""]begin drawing a sigil of the Geometer.")
```

Comment:
A lot of the raw checks for blood volume like this should definitely be `CAN_HAVE_BLOOD` checks